plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPlugin.java

```diff
@@ -40,6 +40,11 @@ public JdbcPlugin(String name, Module module)
     @Override
     public Iterable<ConnectorFactory> getConnectorFactories()
     {
-        return ImmutableList.of(new JdbcConnectorFactory(name, combine(new CredentialProviderModule(), module)));
+        return ImmutableList.of(new JdbcConnectorFactory(
+                name,
+                combine(
+                        new CredentialProviderModule(),
+                        new ExtraCredentialsBasedJdbcIdentityCacheMappingModule(),
```

Comment:
Is this binding "needed" for every connector? Or should each connector module be responsible to install this module themselves?

Reply:
This is an OK change from refactor perspective, as long as all `CredentialProviderModule` usages get updated, but you're right, we could want a different refactor instead, giving each connector more control, and then we could subsequently remove unneeded occurrences.

Reply:
> Is this binding "needed" for every connector? 

It is as this is required for `CachingJdbcClient`, however it is not always a good thing to install this module together with `CredentialProviderModule`